browser.js

```diff
@@ -0,0 +1,125 @@
+const pdf = require("pdfjs");
```

Comment:
Thanks for keeping the interface compatible. This way we can release this as a new feature-version.

The downside of this approach is, that there is a lot of duplicated code. I'm ok with merging it like this but I think we should refactor this to only export one PDFMerger class that support both, local links AND FileBuffers. But this would be a breaking change that should be released as a seperate major version.

Reply:
Yeah. I figured splitting it out was better to start. I'm happy to do the follow on PR to refactor.